editor/src/components/inspector/sections/comment-section.tsx

```diff
@@ -0,0 +1,75 @@
+import '@liveblocks/react-comments/styles.css'
+import React from 'react'
+import { FlexColumn, FlexRow, InspectorSubsectionHeader } from '../../../uuiui'
+import { ClientSideSuspense } from '@liveblocks/react'
+import { Comment } from '@liveblocks/react-comments'
+import { stopPropagation } from '../common/inspector-utils'
+import type { ThreadMetadata } from '../../../../liveblocks.config'
+import { useThreads } from '../../../../liveblocks.config'
+import type { ThreadData } from '@liveblocks/client'
+import { useDispatch } from '../../editor/store/dispatch-context'
+import { canvasPoint, canvasRectangle } from '../../../core/shared/math-utils'
+import { scrollToPosition, switchEditorMode } from '../../editor/actions/action-creators'
+import { EditorModes } from '../../editor/editor-modes'
+
+export const CommentSection = React.memo(() => {
+  return (
+    <div onKeyDown={stopPropagation} onKeyUp={stopPropagation}>
+      <InspectorSubsectionHeader>
+        <FlexRow
+          style={{
+            flexGrow: 1,
+            gap: 8,
+            height: 42,
+          }}
+        >
+          <span>Comments</span>
+        </FlexRow>
+      </InspectorSubsectionHeader>
+      <ClientSideSuspense fallback={<div>Loading…</div>}>
```

Comment:
`MultiplayerWrapper` ?

Reply:
Fixed in 27f9cc54b598ef3c2dcb27709bfe0025b7ce61ae